.env.example

```diff
@@ -8,6 +8,7 @@ VITE_OISY_URL=https://oisy.com
 VITE_ERC20_ICP_SYMBOL=ckICP
 VITE_ERC20_ICP_REPO_URL=https://github.com/dfinity/erc20-icp
 VITE_COINGECKO_API_URL=https://api.coingecko.com/api/v3
+VITE_BLOCKCHAIN_API_URL=https://blockchain.info
```

Comment:
Didn't we already reviewed this in another PR?

Reply:
The branch needed rebasing, unrelated changes are gone now.